packages/asc-ui/src/components/Breadcrumbs/BreadcrumbsStyle.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+import { themeColor } from '../../utils/themeUtils'
+
+export default styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+
+  li:first-child span {
```

Comment:
Not quite sure but since we're using a styled component for this I think you can write:

```suggestion
  li:first-child ${Icon} {
```

Reply:
changed that